c10/cuda/CUDACachingAllocator.cpp

```diff
@@ -1596,17 +1518,45 @@ void notifyCaptureBegin(
       graph_id, mempool_id);
 }
 
-void notifyCaptureEnd(int device, CaptureId_t graph_id) {
+void notifyCaptureAboutToEnd(int device, CaptureId_t graph_id) {
   assertValidDevice(device);
-  caching_allocator.device_allocator[device]->notifyCaptureEnd(graph_id);
+  caching_allocator.device_allocator[device]->notifyCaptureAboutToEnd(graph_id);
 }
 
+void notifyCaptureEnded(int device, CaptureId_t graph_id) {} // no-op
+
 void notifyCaptureDestroy(int device, MempoolId_t mempool_id) {
   assertValidDevice(device);
   caching_allocator.device_allocator[device]->notifyCaptureDestroy(mempool_id);
 }
 
-//
+void* raw_alloc(size_t nbytes) {
```

Comment:
did you change these at all?

Reply:
No, i just moved them above the IPC stuff to organize them alongside the other functions that participate in CUDA_ALLOCATOR_BACKEND_INTERFACE (because the IPC stuff doesn't yet).